sw/device/lib/testing/test_rom/test_rom.c

```diff
@@ -83,6 +84,16 @@ bool rom_test_main(void) {
     }
   }
 #endif
+  // Initialize Ibex cpuctrl (contains icache / security feature enablements).
+  uint32_t cpuctrl_csr;
+  CSR_READ(CSR_REG_CPUCTRL, &cpuctrl_csr);
+  uint32_t cpuctrl_otp_val = abs_mmio_read32(
+      TOP_EARLGREY_OTP_CTRL_CORE_BASE_ADDR + OTP_CTRL_SW_CFG_WINDOW_REG_OFFSET +
+      OTP_CTRL_PARAM_CREATOR_SW_CFG_CPUCTRL_OFFSET);
+  cpuctrl_csr = bitfield_field32_write(
+      cpuctrl_csr, (bitfield_field32_t){.mask = 0x3f, .index = 0},
+      cpuctrl_otp_val);
+  CSR_WRITE(CSR_REG_CPUCTRL, cpuctrl_csr);
```

Comment:
I think this should go in the `ifdef` block above or english breakfast will be sad (it doesn't have OTP IIRC).

Reply:
oops, yes this was supposed to go in the block above. thanks for the reminder! fixed.